bugshaker/src/main/java/com/github/stkent/bugshaker/utilities/SendEmailUtil.java

```diff
@@ -0,0 +1,40 @@
+package com.github.stkent.bugshaker.utilities;
+
+import java.util.List;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.content.pm.ResolveInfo;
+import android.net.Uri;
+import android.support.annotation.NonNull;
+
+import com.github.stkent.bugshaker.flow.email.FeedbackEmailIntentProvider;
+
+public class SendEmailUtil {
```

Comment:
Good job creating this Util. This will allow us to not touch code within the FeedbackEmailFlowManager, so we don't have to take the pains of converting it into a Util.